app/scripts/controllers/meeting.status.ctrl.js

```diff
@@ -27,17 +28,36 @@ angular.module('dashboard')
         }
     }
 
-    $scope.$watch(
-        // This function returns the value being watched. It is called for each turn of the $digest loop
-        function() {
-            return StorageSrv.get(KEY.MEETING);
-        },
-        function(newObject, oldObject) {
-            if (newObject !== oldObject) {
-                self.meeting = newObject;
+    if (meetingItem) {
+        AhjoMeetingSrv.getMeeting(meetingItem.meetingGuid)
+        .then(function(response) {
+            $log.debug("meetingStatusCtrl: getMeeting then:");
+            if (response && response.objects instanceof Array && response.objects.length) {
+                self.meeting = response.objects[0];
+                if (self.meeting && self.meeting.topicList.length) {
+                    var topic = self.meeting.topicList[0];
+                    StorageSrv.set(KEY.TOPIC, topic);
+                }
             }
-        }
-    );
+            else {
+                self.meeting = {};
+                StorageSrv.set(KEY.TOPIC, {});
+            }
+        },
+        function(error) {
+            $log.error("meetingStatusCtrl: getMeeting error: " +JSON.stringify(error));
+            self.error = error;
+        },
+        function(notify) {
+            $log.debug("meetingStatusCtrl: getMeeting notify: " +JSON.stringify(notify));
+        })
+        .finally(function() {
+            $log.debug("meetingStatusCtrl: getMeeting finally: ");
+        });
+    }
+    else {
+        $state.go(APPSTATE.HOME, {menu: MENU.CLOSED});
```

Comment:
Would there be a unified way to `go` to default state defined by router, so that statepareams are taken into acccount? Eg. the state name and default params could be set to a rootscope variable, default state param of course preferably in state definition